pkg/kubelet/images/image_gc_manager.go

```diff
@@ -264,6 +264,11 @@ func (im *realImageGCManager) GarbageCollect() error {
 
 	// If over the max threshold, free enough to place us at the lower threshold.
 	usagePercent := 100 - int(available*100/capacity)
+	if im.policy.HighThresholdPercent == 0 {
+		glog.Warningf("Disk usage percent is %d, the high threshold is 0, will not image GC on device %q at mount point %q", usagePercent, fsInfo.Device, fsInfo.Mountpoint)
+		return nil
+	}
+
```

Comment:
I am not very sure whether this change will make sense.

If we really make an aggrement that when HighThresholdPercent is set to 0, we will not do garbage collection, we should remind this change to users here:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L310

Reply:
if the HighThresholdPercent set to 0, images will be all deleted to free space.
I will add the instructions in the flag.

Reply:
Frankly speaking, I am not particularly supporting this change.
This breaks backward compatible and makes user pain.

Can we figure out other ways to disable this garbage collection? @vishh @ncdc

Reply:
+1. I'd prefer injecting a stub image GC manager instead of adding this code here.

Reply:
@vishh how about add the logic at https://github.com/kubernetes/kubernetes/blob/849d7f859599d4aa327d773b4de4839867523e61/pkg/kubelet/kubelet.go#L1234
when the HighThresholdPercent is set 100%, just stub the imageManager to do garbage collect.

Reply:
@jiaxuanzhou SGTM